src/Features/Core/Portable/CodeRefactorings/ExtractMethod/AbstractExtractMethodCodeRefactoringProvider.cs

```diff
@@ -111,7 +115,14 @@ private static async Task<CodeAction> ExtractLocalFunctionAsync(Document documen
 
             if (localFunctionResult.Succeeded || localFunctionResult.SucceededWithSuggestion)
             {
-                var codeAction = new MyCodeAction(FeaturesResources.Extract_local_function, c => AddRenameAnnotationAsync(localFunctionResult.Document, localFunctionResult.InvocationNameToken, c), nameof(FeaturesResources.Extract_local_function));
+                var codeAction = new MyCodeAction(
+                    FeaturesResources.Extract_local_function,
+                    async c =>
+                    {
+                        var (document, invocationNameToken) = await localFunctionResult.GetFormattedDocumentAsync(c).ConfigureAwait(false);
```

Comment:
i see how this pushed formatting till later.  i'm not seeing how we pushed simplification till later.  can you clarify?

Reply:
ignore me.  github had not loaded full pr yet.